<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nota de Débito</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>521</width>
      <height>481</height>
     </rect>
    </property>
    <property name="title">
     <string>Dados da ETS / Dados do Cliente / Dados da Nota</string>
    </property>
    <widget class="QPushButton" name="btnCancelar">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>440</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancelar</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icones/document_1204109.png</normaloff>icones/document_1204109.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSalvarCredor">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>440</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Salvar</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icones/floppy-disk_1311670.png</normaloff>icones/floppy-disk_1311670.png</iconset>
     </property>
    </widget>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>501</width>
       <height>411</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Dados  da ETS</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>491</width>
         <height>191</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>471</width>
          <height>171</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QLineEdit" name="txtCnpj">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>42</y>
           <width>133</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Cnpj</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="txtRazao">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>40</y>
           <width>201</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>20</y>
           <width>101</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Razão social</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btnConsultar">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>41</y>
           <width>91</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Consultar </string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icones/magnifying-glass_2703414.png</normaloff>icones/magnifying-glass_2703414.png</iconset>
         </property>
        </widget>
        <widget class="QLineEdit" name="txtEndereco">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>161</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Endereço</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="txtNumero">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>90</y>
           <width>51</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>70</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Numero</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>70</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Bairro</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="txtBairro">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>90</y>
           <width>201</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>120</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Cidade</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="txtCidade">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>140</y>
           <width>111</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="txtEstado">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>140</y>
           <width>111</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>120</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Estado</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Dados do Cliente</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>491</width>
         <height>191</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>471</width>
          <height>171</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QLineEdit" name="txtCnpj_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>42</y>
           <width>133</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Cnpj</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="txtRazao_2">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>40</y>
           <width>201</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>20</y>
           <width>101</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Razão social</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btnConsultar_2">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>41</y>
           <width>91</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Consultar </string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icones/magnifying-glass_2703414.png</normaloff>icones/magnifying-glass_2703414.png</iconset>
         </property>
        </widget>
        <widget class="QLineEdit" name="txtEndereco_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>161</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Endereço</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="txtNumero_2">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>90</y>
           <width>51</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>70</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Numero</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_12">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>70</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Bairro</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="txtBairro_2">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>90</y>
           <width>201</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_13">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>120</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Cidade</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="txtCidade_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>140</y>
           <width>111</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="txtEstado_2">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>140</y>
           <width>111</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_14">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>120</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Estado</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Dados da Nota</string>
      </attribute>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
